compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4516,7 +4522,7 @@ private STNode parseExpressionRhs(SyntaxKind tokenKind, OperatorPrecedence curre
             return lhsExpr;
         }
 
-        if (lhsExpr.kind == SyntaxKind.ASYNC_SEND_ACTION) {
+        if (lhsExpr != null && lhsExpr.kind == SyntaxKind.ASYNC_SEND_ACTION) {
```

Comment:
Can lhsExpr be null?

Reply:
That is from the previous commit. Now we don't need that. I'll correct.